src/main/index.ts

```diff
@@ -56,8 +57,8 @@ module.exports = NodeHelper.create({
             if (!fs.existsSync(outputPath)) {
                 fs
                     .createReadStream(
-                        path.resolve(__dirname, "resources/alexa/sorry-im-not-sure.mpeg"),
-                    )
+                    path.resolve(__dirname, "resources/alexa/sorry-im-not-sure.mpeg"),
```

Comment:
Are you using Prettier plugin in VSCode now? Because when I formatted this file it reverted this change :D

Reply:
Ah I am not, I will download it now